portfolio/src/test/java/com/google/sps/DataServletTest.java

```diff
@@ -12,45 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps;
+package com.google.sps.servlets;
```

Comment:
Since you changed the package you need to move the file to the `servlets` directory too.

Reply:
ah ok... that makes sense. Move it to the same location as the other Java files?